src/stores/models/Vote.js

```diff
@@ -0,0 +1,42 @@
+import { TransactionStatus } from 'constants';
+
+import { satoshiToDecimal, decimalToSatoshi } from '../../helpers/utility';
+
+/**
+ * Model for Transactions.
+ */
+export default class Vote {
+  type = '' // One of: DBVoteType
+  txid = '' // Transaction ID assigned by the blockchain
+  status = '' // only success
```

Comment:
lets not add this field to the model itself, but add it to the UI logic. someone might confuse that the DB Vote model has a `status` field when it doesn't so lets just fill in the `Success` field in table if its not a `Transaction` type.

Reply:
for data from `transactions.db`, do we display it if it's status of fail?

Reply:
Yes we do show failed txs as well.